news/2021-01-14-performance-points-updates.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "Performance Points Updates"
+date: 2021-01-14 10:00:00 +0000
+---
+
+Thanks to the effort of a number of public contributors, we're excited to announce a new set of changes to osu!'s performance points (pp) algorithm that should help freshen things up at high levels of play. Read on for more details!
+
+![](/wiki/shared/news/2021-01-14-performance-points-updates/star-rating.jpg)
+
+Set off during the last few days of 2020, these new incoming changes represent an effort from various members of the community to try and address long-standing issues with one of the game's most important progression systems.
+
+For those unfamiliar with the technical minutiae of the system, please consult [the wiki article](/wiki/Performance_points) for an overview of the general ideas involved. 
+
+To summarize the article very briefly (you should really read the whole thing on your own first!), pp encompasses several core values as follows:
+
+- **Aim**, which considers how difficult it is to consistently hit notes back-to-back during play (affected by things like approach rate, jumps, Flashlight, Hidden, Hard Rock and more)
+- **Speed**, which considers how quickly new elements are presented to the player during the course of the map (affected by things like the number of hit objects over time, BPM, Double Time and Half Time)
+- **Accuracy**, which considers the player's ability to stay on time rhythmically and how difficult this is to achieve (affected by things like Hidden, Hard Rock, Flashlight and more)
+- **Strain**, which considers how often and how long a player has to endure intense or technically demanding patterns during play (affected by things like streams, fast jumps and more)
```

Comment:
```suggestion
- **Aim**, which considers how difficult it is to consistently hit notes back-to-back during play (affected by things like approach rate, jumps, Flashlight, Hidden, Hard Rock, and more)
- **Speed**, which considers how quickly new elements are presented to the player during the course of the map (affected by things like the number of hit objects over time, BPM, Double Time, and Half Time)
- **Accuracy**, which considers the player's ability to stay on time rhythmically and how difficult this is to achieve (affected by things like Hidden, Hard Rock, Flashlight, and more)
- **Strain**, which considers how often and how long a player has to endure intensely or technically demanding patterns during play (affected by things like streams, fast jumps, and more)
```
minor formatting fixes

Reply:
intense is correct; commas not being there are correct (stylistic choice, see https://en.wikipedia.org/wiki/Serial_comma)